app/src/main/res/values/strings.xml

```diff
@@ -646,7 +646,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="back">Back</string>
   <string name="welcome_custom_picture_selector_text">Welcome to Custom Picture Selector</string>
   <string name="custom_selector_info_text1">This picker shows differently pictures that are already to Commons.</string>
-  <string name="custom_selector_info_text2">Unlike the picture on the left, the picture on the right has the Commons logo indicating it is already uploaded.</string>
+  <string name="custom_selector_info_text2">Unlike the picture on the left, the picture on the right has the Commons logo indicating it is already uploaded. \n Hold tap for image preview.</string>
```

Comment:
I think `Touch & hold` is the most common term? https://support.google.com/accessibility/android/answer/6006989?hl=en

Reply:
Done.